README.md

```diff
@@ -73,16 +73,27 @@ This diagram covers the major classes of transformations. The most basic ones ar
 Here is a very simple example of an algebra (`eval`) and how to apply it to a recursive structure.
 
 ```scala
+
+// we will need a Functor[Expr] in order to call embedT bellow
```

Comment:
This should say `embed` instead of `embedT` now (and there’s an extra blank line above it)